A76-ring-hash-improvements.md

```diff
@@ -0,0 +1,169 @@
+A76: Improvements to the Ring Hash LB Policy
+----
+* Author(s): @atollena
+* Approver: ?
+* Status: Draft
+* Implemented in: <language, ...>
+* Last updated: 2024-01-15
+* Discussion at: TODO
```

Comment:
Please start a mailing list thread and add a link to it here, as per the gRFC process.

Reply:
Done.